src/pages/home/report/ReportActionCompose.js

```diff
@@ -503,12 +415,7 @@ class ReportActionCompose extends React.Component {
 
         // Prevents focusing and showing the keyboard while the drawer is covering the chat.
         const isComposeDisabled = this.props.isDrawerOpen && this.props.isSmallScreenWidth;
-        const isConciergeChat = this.props.report.participants
-            && _.contains(this.props.report.participants, CONST.EMAIL.CONCIERGE);
-        let isBlockedFromConcierge = false;
-        if (isConciergeChat && !_.isEmpty(this.props.blockedFromConcierge)) {
-            isBlockedFromConcierge = User.isBlockedFromConcierge(this.props.blockedFromConcierge.expiresAt);
-        }
+        const isBlockedFromConcierge = ReportUtils.isBlockedFromConciergeChat(this.props.report, this.props.blockedFromConcierge);
```

Comment:
I recommend we set this the same way I recommended above. Instead of creating this new `ReportUtils.isBlockedFromConciergeChat` function.

Reply:
Updated